src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+/* eslint-disable no-console */
+import classNames from 'classnames';
+import { Todo } from '../../types/TodoType';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+  largestId: number;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo, largestId }) => {
+  return (
+    <article
+      data-id={largestId + 1}
```

Comment:
Why we need `largestId` and can't simple use `todo.id`? Try not to pass `largestId` prop it seems strange

Reply:
![image](https://user-images.githubusercontent.com/105390355/219865785-c6fe111e-06ec-4113-9c86-bcf32dcd07fe.png)

As I understand it, I need to find the largest id in the list of all cases and add 1 to it, and then assign each component to the date id. Right? If you assign todo.id, then all numbers will be different and the test will fail.